### PR TITLE
fixed #7

### LINKED ISSUE #7: when page scrolled, the selectionBox will out of the selection
issue

### PR DESCRIPTION
**Pull request by FrancisKong**

### CODE REVIEW

**Comment by FrancisKong** [comment]
I don't know what's wrong  with this webpack configuration, the built file can't import to projects, trying fix that.

**Comment by FrancisKong** [comment]
Bug fixed!

File: src/DragSelect.vue [hunk 1]
```diff
@@ -65,6 +66,19 @@
       }
     },
     methods: {
+      getScroll () {
+        const body = {
+          top: document.body.scrollTop > 0 ? document.body.scrollTop : document.documentElement.scrollTop,
```
[the remark this answers is not in the document]
Comment (FrancisKong) [reply src/DragSelect.vue]: Nice code!

**Comment by FrancisKong** [comment]
@stephan281094 

```
24:29-39 "export 'default' (imported as 'DragSelect') was not found in '../../dist/vue-drag-select.js'
```

This error occurs because you can't use `import` and `module.exports` at same time, and you need to move built file into the `node_modules` folder or exclude the file from `webpack`

**Comment by stephan281094** [comment]
Sorry for late response. I've re-added `vue-no-ssr` to the example, since it was still having some issues with SSR.

Thanks again for the great work!

**Pull request merged**